src/main/java/net/logstash/logback/composite/CompositeJsonFormatter.java

```diff
@@ -187,9 +263,6 @@ private JsonFactory decorateFactory(JsonFactory factory) {
     }
     
     protected void writeEventToGenerator(JsonGenerator generator, Event event) throws IOException {
-        if (!isStarted()) {
-            throw new IllegalStateException("Encoding attempted before starting.");
-        }
```

Comment:
Why is this removed?

Reply:
This method is called by `JsonFormatter#writeEvent` which in turn is called by `this#writeEvent` and that method is already guarded by the same condition.